ggml.c

```diff
@@ -4566,6 +4570,10 @@ struct ggml_context * ggml_init(struct ggml_init_params params) {
         /*.objects_end        =*/ NULL,
         /*.scratch            =*/ { 0, 0, NULL, },
         /*.scratch_save       =*/ { 0, 0, NULL, },
+#ifdef GGML_RECOVERABLE_ERRORS
+        /*.last_error_code    =*/ GGML_ERRCODE_SUCCESS,
+        /*.last_error_msg     =*/ { 0 },
+#endif
```

Comment:
Should have the same struct contents regardless if `GGML_RECOVERABLE_ERRORS` is defined.
Same for `ggml_context`

Reply:
Apologies, I actually missed some details in my changes yesterday. Sorry for wasting your time if you already checked this. It should be fixed now and I will be more careful in the future.